### ISSUE TITLE
Enhancement: Limit the creation of Tags to admins

### ISSUE THREAD

**Issue by simondavies** [laraveluk/website#85]
At the moment when creating a post, a user can add as many tags as they like and also can name them anything.

I propose that we switch the tags to a select dropdown, where a user can then select the tags they want, and as many as they like, or also limit them to 3.

Similar to StackOverflow, we can eventually add a point system in where they can then earn the right to add tags etc.

The idea of this is to prevent the saturation and the number of tags that are created. 

Like to know your thoughts

**Comment by stevepop** [comment]
@simondavies, I agree with this suggestion. We may end up with too many tags which could even slow down the fetching of Posts since with each Post we fetch the tags as well. Good points! Will look into this.

**Comment by simondavies** [comment]
no worries, let me know and can look into these if you like
